exporters/variables-scss/src/index.ts

```diff
@@ -41,45 +36,46 @@ Pulsar.export(async (sdk: Supernova, context: PulsarContext): Promise<Array<AnyO
       tokens = await sdk.tokens.computeTokensByApplyingThemes(tokens, [currentTheme]);
     } else {
       // Don't allow applying theme which doesn't exist in the system
-      throw new Error("Unable to apply theme which doesn't exist in the system.");
+      throw new Error('Unable to apply theme which does not exist in the system.');
     }
   }
 
   // Convert all color tokens to CSS variables
-  const simpleTokens = generateSimple(tokens);
-  const mappedTokens = new Map(simpleTokens.map((token) => [token.id, token]));
+  const mappedTokens = new Map(tokens.map((token) => [token.id, token]));
 
-  const cssVariables = tokens
-    .filter((t) => t.tokenType === TokenType.color)
-    .map((token) => colorTokenToCSS(token as ColorToken, mappedTokens, tokenGroups))
-    .join('\n');
+  const createTextFile = (relativePath: string, fileName: string, content: string): OutputTextFile => {
+    return FileHelper.createTextFile({
+      relativePath,
+      fileName,
+      content,
+    });
+  };
 
-  // Create CSS file content
-  let content = cssVariables;
-  if (exportConfiguration.generateDisclaimer) {
-    // Add disclaimer to every file if enabled
-    content = `/* This file was generated by Supernova, don't change by hand */\n${content}`;
-  }
+  const files = generateFiles(tokens, mappedTokens, tokenGroups);
 
-  // Create output file and return it
   return [
-    FileHelper.createTextFile({
-      relativePath: './',
-      fileName: '_colors.scss',
-      content,
-    }),
-    FileHelper.createTextFile({
-      relativePath: './original-data/',
-      fileName: '_colors.json',
-      content: JSON.stringify(tokens, null, 2),
+    ...files.map((file) => {
+      return createTextFile('./global/', file.fileName, file.content);
     }),
+    // only for debugging purposes
+    createTextFile(
+      './original-data/',
+      '_original-tokens.json',
+      JSON.stringify(
+        tokens.map((token) => ({
+          tokenType: token.tokenType,
+          // @ts-ignore-next-line
```

Comment:
Missing description for this comment why we are ignoring this. And same for the comment few lines lower.